src/main/webapp/app/components/data-table/data-table.component.ts

```diff
@@ -159,7 +179,13 @@ export class DataTableComponent implements OnInit, OnChanges {
      */
     private getCachedEntitiesPerPage = () => {
         const cachedValue = localStorage.getItem(this.perPageCacheKey);
-        return cachedValue ? parseInt(cachedValue, 10) : this.DEFAULT_PAGING_VALUE;
```

Comment:
Just saw this now, could you please switch to ngx-webstorage?

Reply:
Thanks for pointing it out, I didn't know about `ngx-webstorage`. It is now being used.